recipe/conda_build_config.yaml

```diff
@@ -0,0 +1,2 @@
+channel_targets:
+- conda-forge ipython_beta
```

Comment:
Just for :eyes: this is an important change on this PR for getting a new channel. This will be served with (i think):


```bash
mamba install -c conda-forge -c conda-forge/label/ipython_beta ipython=8.0.0b1
```

Reply:
If we want to start getting bot ABI PRs, we'd need to add the `8.x-beta` branch to:

https://github.com/conda-forge/ipython-feedstock/blob/master/conda-forge.yml#L3